packages/x-utils/tsconfig.eslint.json

```diff
@@ -2,6 +2,5 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "noEmit": true
-  },
-  "exclude": []
+  }
```

Comment:
After removing this `exclude` property everything seems to be working as expected